src/graphql/posts/mutations.ts

```diff
@@ -21,7 +21,6 @@ const PostMutations = {
   removePost: async (_: any, { input }: {input: RemovePostInputType}) => {
     const PostModel = getPostModel()
```

Comment:
just in case you haven't planned to,  it's a good idea to put database logic in its own Data source classes so that the code in here is light.

Reply:
I'm planning to do that in the future. I wanted to start out by keeping it simple.